src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
+ * TYPES
+ */
+
+type node = {
+  x: Number;
+  y: Number;
+  value: Number;
+  isExit: Boolean;
+};
+
+type edge = {
+  from: string;
+  to: string;
+  direction: string;
```

Comment:
use an enum instead

Reply:
Enum is used with the [commit 82e57ce ](https://github.com/marmelab/kuba-integration/commit/82e57ce3a40dd7ed5da9fee6880050c75b8caa4b) change